chapters/en/chapter10/5.mdx

```diff
@@ -1 +1,362 @@
+
+Since our audio input is continuous, we can't represent an amplitude value for every possible time step. Instead, we _sample_ amplitude values at fixed time steps.
+
+The interval with which we sample our audio is known as the _sampling rate_. For our dataset, we can see that the sampling rate is 16000, meaning 16000 amplitude values are provided each second. Keep the sampling rate in the back of your mind, it'll be important when we come to processing our data later on!
+
+Whilst 1-dimensional arrays are a suitable format for machines, they're not much use to us as humans! We want to be able to **listen** to our audio to get a feel for the speech and recording conditions. Here, we have two options:
+
+1. Convert our 1-dimensional array into a human-friendly format (mp3 or wav).
+2. Look to the Hugging Face Hub!
+
+Option 2 seems much easier to me! Let's head over to the LibriSpeech ASR dataset card on the Hugging Face Hub: https://huggingface.co/datasets/librispeech_asr
+
+Slap bang in the middle of the dataset card we have exactly what we're looking for: the dataset viewer! The dataset viewer shows us the first 100 samples of any dataset. What's more, it's loaded up the audio samples ready for us to listen to in real-time! If we hit the play button on the first sample, we can listen to the audio and see the corresponding text. Have a scroll through the samples for the train and test sets to get a better feel for the audio data that we're dealing with. You’ll notice how clear the audio is and how well spoken the sentences are. This provides us with a clue as to why the top models do so well on LibriSpeech! The audio conditions are very conducive to high system performance.
+
+### Models for speech recognition
+
+We can decompose speech recognition models into two parts:
+1. Encoder: an acoustic model that maps the raw audio input into a sequence of hidden-states.
+
+diagram
+
+2. Decoder: maps the sequence of hidden-states to logits over the vocabulary.
+
+diagram
+
+The encoder is typically of Wav2Vec2 architecture, introduced in Section 4 of this Chapter. We have some flexibility in our choice of decoder. We could either use a simple linear layer that maps the Wav2Vec2 hidden states directly to output logits over our vocabulary. Or, we could pair our Wav2Vec2 encoder with a decoder model, giving a speech encoder-decoder style model, analogous to the NLP encoder-decoder model introduced in Chapter 1.
+
+A simple linear layer will give a smaller, faster overall model, but will be more susceptible to spelling and grammatical errors. Adding a decoder model greatly improves the quality of transcriptions, at the cost of a larger, slower model. This is because the decoder model is pre-trained on a large corpus of text, enabling us to leverage its learned text representations.
```

Comment:
```suggestion
A simple linear layer will give a smaller, faster overall model but will be more susceptible to spelling and grammatical errors. Adding a decoder model greatly improves the quality of transcriptions because it is pre-trained on a large corpus of text, enabling us to leverage its learned text representations. The tradeoff is a larger, slower model.
```